content/en/docs/gke/deploy/delete-cli.md

```diff
@@ -25,36 +20,19 @@ This guide assumes the following settings:
   export KF_DIR=<path to your Kubeflow application directory>
   ``` 
 
-* The `${CONFIG_FILE}` environment variable contains the path to your 
-  Kubeflow configuration file.
-
-  ```
-  export CONFIG_FILE=${KF_DIR}/{{% config-file-gcp-iap %}}
-  ```
-
-    Or:
+## Deleting your deployment
 
-  ```
-  export CONFIG_FILE=${KF_DIR}/{{% config-file-gcp-basic-auth %}}
-  ```
 
-For further background about the above settings, see the guide to
-[deploying Kubeflow with the CLI](/docs/gke/deploy/deploy-cli).
+1. To just delete the applications running in the Kubeflow namespace just delete that namespace
 
-## Deleting your deployment
+   ```
+   kubectl delete namespace kubeflow
+   ```
 
-Run the following commands to delete your deployment and reclaim all GCP
-resources:
+1. To delete the cluster and all GCP resources
```

Comment:
Suggestion: 

- Add an "call to action" here to bring this in line with a lot of Google/Microsoft docs:

```markdown
1. To delete the cluster and all GCP resources, run the following commands
```

(You can use colons—":"—at the end with each example too)